src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ImportCompletionItem.cs

```diff
@@ -219,12 +220,7 @@ private static (ISymbol? symbol, int overloadCount) GetSymbolAndOverloadCount(Co
     public static CompletionItem MarkItemToAlwaysFullyQualify(CompletionItem item)
     {
         var itemProperties = item.GetProperties();
-
-        using var _ = ArrayBuilder<KeyValuePair<string, string>>.GetInstance(itemProperties.Length + 1, out var builder);
-        builder.AddRange(itemProperties);
-        builder.Add(new KeyValuePair<string, string>(AlwaysFullyQualifyKey, AlwaysFullyQualifyKey));
-
-        return item.WithProperties(builder.ToImmutable());
+        return item.WithProperties([.. itemProperties, KeyValuePairUtil.Create(AlwaysFullyQualifyKey, AlwaysFullyQualifyKey)]);
```

Comment:
collection exprs work nicely when the inputs are fixed length, and we're not transforming anything.